auto_asr/subtitle_processing/processors/translate.py

```
@@ -0,0 +1,72 @@
+        results: dict[str, str] = {}
+
+        def run_batch(payload: dict[str, str]) -> dict[str, str]:
+            return ctx.chat_json(system_prompt=system_prompt, payload=payload)
+
+        with ThreadPoolExecutor(max_workers=concurrency) as ex:
+            futs = [ex.submit(run_batch, b) for b in batches]
+            for fut, batch in zip(futs, batches, strict=False):
+                try:
+                    results.update({str(k): str(v) for k, v in fut.result().items()})
+                except Exception as e:
+                    logger.exception("translate batch failed, fallback to original: %s", e)
```

Ruff (0.14.14)

Redundant exception object included in `logging.exception` call

(TRY401)